ONSdigital/rm-action-service
# Rerun Aborted Actions

# Motivation and Context
On some occasions actions can be aborted in the action plan that at a later date need to be rerun 
(eg. a missing template in an action email).
This would require the API PR to go in for the new state transition added: 

# What has changed
New state transition added allowing us to go from `ABORTED` -> `SUBMITTED`
New endpoint which allows us to re run aborted actions and updated tests while adding new ones.

# How to test?
- Update an entry in the `action` table to the status of `ABORTED` and hit the new endpoint using the corresponding `id` as a query param and check that that state of action has transitioned from `ABORTED` -> `SUBMITTED`.
- Check builds correctly and tests pass.

# Links
Card: https://trello.com/c/2n3BYl7R/422-rerun-aborted-actions-l
